src/workspace.jl

```diff
@@ -41,7 +42,7 @@ wstype(mod, name, ::Symbol) = "tag"
 wstype(mod, name, ::Exception) = "mixin"
 wstype(mod, name, ::Undefined) = "ignored"
 
-wsicon(mod, name, val) = isconst(mod, name) ? "c" : "v"
+wsicon(mod, name, @nospecialize(val)) = isconst(mod, name) ? "c" : "v"
```

Comment:
I believe those uses of `@nospecialize` would be good here, but still want you to confirm that.

Reply:
Yeah, makes sense to have this.

Reply:
Thanks for checking this !